src/main/java/seedu/nutrigoals/model/user/Age.java

```diff
@@ -7,9 +7,11 @@
  * Represents the user's age
  */
 public class Age {
-    public static final String VALIDATION_REGEX = "^[1-9]\\d*$";
-    public static final String MESSAGE_CONSTRAINTS = "Age must be a number greater than 0";
+    public static final String VALIDATION_REGEX = "\\d+";
+    public static final String MESSAGE_CONSTRAINTS = "Age must take on a positive integer than is not too large.";
```

Comment:
Would this error message be too vague to the user? Especially since there is now also a lower bound restriction on the age? Could consider explicitly stating the upper and lower bounds on the value !

Reply:
Fixed!